composer.json

```diff
@@ -16,15 +16,15 @@
         }
     ],
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.1",
```

Comment:
Not sure if we can drop that on next minor release.

Reply:
We can, and did on all other packages.